Dockerfile.client

```diff
@@ -0,0 +1,17 @@
+FROM centos:centos8
+ENTRYPOINT [ "/usr/local/bin/osg-notify" ]
+
+LABEL maintainer="OSG Software <help@opensciencegrid.org>"
+LABEL name="OSG 3.5 OSG-Notify client"
+
+RUN yum -y install https://repo.opensciencegrid.org/osg/3.5/osg-3.5-el8-release-latest.rpm \
+                   epel-release \
+                   yum-utils
+
+RUN yum -y install python3 \
+                   python3-requests \
+                   python3-gnupg
+
+COPY topology/bin/osg-notify               /usr/local/bin
+COPY topology/src/net_name_addr_utils.py   /usr/lib/python3.6/site-packages
+COPY topology/src/topology_utils.py        /usr/lib/python3.6/site-packages
```

Comment:
You should specify that the ENTRYPOINT is /usr/local/bin/osg-notify, so that is the command that will be run when people run the container. Use ENTRYPOINT (specifically, the 'exec form' of it) instead of CMD because people will need to pass arguments to the container.

https://docs.docker.com/engine/reference/builder/#entrypoint

Reply:
How would I go about testing to make sure it does all of these correctly? I have made some changes but would like to test more before pushing.

Reply:
My mistake, I didn't see that line. Usually ENTRYPOINT lines are near the end of the file.